### PR TITLE
Added week 8 event

### PR DESCRIPTION
**Pull request by gitbeanz**
### Description
Added a week 8 event on typography and color theory

### Testing
Testing for responsiveness and ensured website was not broken. 

image

### CODE REVIEW

**Review by ryqndev — APPROVED** [review]
lgtm! ty

File: src/assets/data/events/all.json [hunk 1]
```diff
@@ -1,12 +1,23 @@
-[{
+[
+  {
+    "title": "Typography and Color Theory",
+    "time": "5/22/2023 8:00 PM",
+    "duration": "120",
+    "type": "Workshop",
+    "desc": "Ever wondered how the iconic New York subway signs came to be? Or how to add your style to any font in the world? Join us in SSL228 to learn about typography in graphic design via Adobe Illustrator! This workshop is for users with all skill levels. Please make sure to have Adobe Illustrator downloaded on your computer before the event! The Adobe Creative Suite is free to all UCI students.",
+    "place": "SSL 228",
```
Comment (taesungh) [inline src/assets/data/events/all.json:8]: Nitpick: consistency in location formatting, would prefer space between building and room number.

**Review by taesungh — APPROVED** [review]
Looks good, please **squash** during merge.

**Comment by taesungh** [comment]
Merged in dec23ad

**Pull request closed without merging**